src/InputGroup/InputGroup.js

```diff
@@ -87,6 +89,7 @@ export class InputGroup extends Component {
                 );
 
                 const inputNumberClasses = classnames(
+                    className,
```

Comment:
It looks like this should be a different prop for the `input` className versus the `div` className. Maybe we should have `className` that get applied to the parent `div` and an `inputClassName` that gets applied to the `input` element.